src/CssInliner.php

```diff
@@ -1044,7 +1049,7 @@ private function removeImportantAnnotationFromNodeInlineStyle(\DOMElement $node)
         $importantStyleDeclarations = [];
         foreach ($inlineStyleDeclarations as $property => $value) {
             if ($this->attributeValueIsImportant($value)) {
-                $importantStyleDeclarations[$property] = \trim(\str_replace('!important', '', $value));
+                $importantStyleDeclarations[$property] = $this->pregReplace('/\\s*+!\\s*+important$/i', '', $value);
```

Comment:
Psalm says that `preg_replace` might return `null`.  I think it's more likely that the server will crash and it won't return anything at all.

Please reject this change and either remove Psalm from the project or tell me how I'm supposed to actually make code changes that satisfy Psalm in some kind-of semi-positive way.

Reply:
In this case, Psalm is right. [`preg_replace` will return `null` if there is an error](https://www.php.net/manual/en/function.preg-replace.php).

For doing this in a cleaner way, we could do something like this:
`… = \preg_replace(…) ?? ''`
This will return an empty string in case of an error.
Or we could throw an exception if `null` is returned to make the problem visible.